lib/oban/worker.ex

```diff
@@ -336,4 +338,18 @@ defmodule Oban.Worker do
   defp validate_opt!(option) do
     raise ArgumentError, "unknown option provided #{inspect(option)}"
   end
+
+  @doc """
+  Return a string representation of a worker module.any()
```

Comment:
This is a perfect fit for a doctest:

```suggestion
  Return a string representation of a worker module. This is particularly useful for normalizing
  worker names when building custom Ecto queries.

  ## Examples
  
  iex> Oban.Worker.to_string(MyApp.SomeWorker)
  "MyApp.SomeWorker"
  
  iex> Oban.Worker.to_string("Elixir.MyApp.SomeWorker")
  "MyApp.SomeWorker"
```

Reply:
Oh, I didn't know it would work without declaring the module. Added!